preview/MsixCore/msixmgrLib/PackageManager.cpp

```diff
@@ -275,7 +275,17 @@ HRESULT PackageManager::GetMsixPackageInfo(const wstring & msixFullPath, shared_
     shared_ptr<Package> packageInfo;
     ComPtr<IStream> packageStream;
     RETURN_IF_FAILED(CreateStreamOnPackageUrl(msixFullPath.c_str(), &packageStream));
-    RETURN_IF_FAILED(PopulatePackageInfo::GetPackageInfoFromPackage(packageStream.Get(), MSIX_VALIDATION_OPTION::MSIX_VALIDATION_OPTION_FULL, &packageInfo));
+    HRESULT hrGetPackageInfoFromPackage = PopulatePackageInfo::GetPackageInfoFromPackage(packageStream.Get(), MSIX_VALIDATION_OPTION::MSIX_VALIDATION_OPTION_FULL, &packageInfo);
+    if (FAILED(hrGetPackageInfoFromPackage) && hrGetPackageInfoFromPackage == 0x8bad0031)

```

Comment:
>0x8bad0031 [](start = 78, length = 10)

MSIX::Error::MissingAppxSignatureP7X

Reply:
kevin just stopped by and the ones i was thinking about were SignatureInvalid   and  CertNotTrusted              

---
In reply to: [351061565](https://github.com/microsoft/msix-packaging/pull/279#discussion_r351061565) [](ancestors = 351061565)

Reply:
Modified to check with MSIX::Error::MissingAppxSignatureP7X.